packages/model-viewer/src/model-viewer-base.ts

```diff
@@ -61,6 +61,14 @@ export const $progressTracker = Symbol('progressTracker');
 export const $getLoaded = Symbol('getLoaded');
 export const $getModelIsVisible = Symbol('getModelIsVisible');
 
+export class Vector3D {
```

Comment:
Let's make this an interface, not a class. Class construction has extra cost associated with it that isn't really called for for such a simple type.